src/test/java/step2/calculator/Expression.java

```diff
@@ -0,0 +1,28 @@
+package step2.calculator;
+
+import java.util.Arrays;
+import java.util.function.BiFunction;
+
+public enum Expression {
+    PLUS("+", (a, b) -> a + b),
+    MINUS("-", (a, b) -> a - b),
+    MULTIPLY("*", (a, b) -> a * b),
+    DIVIDE("/", (a, b) -> a / b);
+
+    private String label;
+    private BiFunction<Integer, Integer, Integer> calculate;
+
+    Expression(String label, BiFunction<Integer, Integer, Integer> calculate) {
+        this.label = label;
+        this.calculate = calculate;
+    }
+
+    static public Expression of(String label) {
+        return Arrays.stream(values()).filter(expression -> expression.label.equals(label)).findFirst().orElseThrow(IllegalArgumentException::new);
```

Comment:
```suggestion
        return Arrays.stream(values())
                .filter(expression -> expression.label.equals(label))
                .findFirst()
                .orElseThrow(IllegalArgumentException::new);
```
Stream을 잘 사용해주셨네요 👍 
다만 한 줄에 한 개의 Stream 메소드를 사용할 수 있도록 해주세요!
그래야 가독성이 좋답니다~

Reply:
expression.label끼리 일치하는지 확인하는 부분도 메소드로 분리해서 관리해주면 어떨까요?